msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -47,7 +47,8 @@
 		var elems = document.getElementsByTagName('sakai-rubric-association');
 		var forumAssignments = document.getElementById("revise:forum_assignments");
 		const createTaskGroup = document.getElementById("revise:createTaskGroup");
-		if (forumAssignments?.value && forumAssignments.value !== "Default_0") {
+		const isTasksWidgetAvailable = !!document.querySelector("sakai-tasks");
+		if (forumAssignments?.value && forumAssignments.value !== "Default_0" && isTasksWidgetAvailable) {
```

Comment:
Can forumAssignments be nullish here? If so, it may just make sense to return at the top of the method instead of using these if blocks.

Reply:
what about `forumAssignments?`  That operator is checking for null

Reply:
If forumAssigments is crucial in this block of code, then why not exit the method early so no more dom queries have to happen. It would make the code clearer too, probably. Also, you can directly compare the optional chaining output using !==. undefined is a primitive so can be used in the equality statements.